<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseExportDialog</class>
 <widget class="QWidget" name="ChooseExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QPushButton" name="notUpgradeBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>318</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cancel export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delegateBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>259</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Encrypted export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upgradeBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Common export</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
